pkg/authenticator/k8s/authenticator.go

```diff
@@ -423,3 +424,17 @@ func consumeInjectClientCertError(path string) string {
 
 	return string(content)
 }
+
+func logIPS(host string) {
+	ips, lookUpError := net.LookupIP(host)
+	if lookUpError != nil {
+		fmt.Fprintf(os.Stderr, "Could not get IPs: %v\n", lookUpError)
+		os.Exit(1)
+	}
+	ipsList := " IP Addresses: "
+	for _, ip := range ips {
+		ipsList += ip.String()
```

Comment:
If you remain in this way maybe this two lines can be combined to one with string formatting